gooey/internal/scrollbar.lua

```diff
@@ -22,13 +22,15 @@ function SCROLLBAR.scroll_to(scrollbar, x, y)
 		local adjusted_height = (scrollbar.bounds_size.y - scrollbar.handle_size.y)
 		local offset_y = scrollbar.vertical and (scrollbar.handle_size.y / 2) or 0
 		handle_pos.y = offset_y + adjusted_height - y * adjusted_height
+		scrollbar.scroll.y = y
 	else
 		local adjusted_width = (scrollbar.bounds_size.x - scrollbar.handle_size.x)
 		local offset_x = scrollbar.vertical and 0 or (scrollbar.handle_size.x / 2)
 		handle_pos.x = offset_x + adjusted_width - x * adjusted_width
+		scrollbar.scroll.x = x
 	end
 	gui.set_position(scrollbar.node, handle_pos)
-	scrollbar.scroll.y = y
+	scrollbar.refresh()
```

Comment:
*thought*: We should be refreshing on `scroll_to`, just like we do on things like `radio#set_selected` or `checkbox#set_checked`, so that the consumer can refresh their styles based on the new scroll value.